docs/reference.rst

```diff
@@ -891,3 +891,45 @@ Stores the number of milliseconds to wait between request retries.  By default,
 
 In the case that both this environment variable and the config's ``rpc_attempt_interval`` property are set, the value in ``rpc_attempt_interal`` will be used.
 
+************
+Localization
+************
+
+Shotgun API offers the ability to return display namnes in user's language.
```

Comment:
*The* Shotgun API

Reply:
How does the API know what the user's language is? I think this is worth explaining. Is it based on the user's language preferences in Shotgun? (What happens with script/API users?)

Reply:
*namnes* (typo)

Reply:
suggestion: return display namnes in user's language -> return field display names in the current user's language.

Reply:
Thanks for all the suggestions ! I updated the documentation.